src/components/services/artistApi.js

```diff
@@ -0,0 +1,37 @@
+export const getArtists = ({ page, search }) => {
+  return fetch(`http://musicbrainz.org/ws/2/artist?query=${search}&fmt=json&offset=${page}`, {
```

Comment:
This is causing y'all to miss the top-listed artist on every search. That's why you never get the real band you search for.
```suggestion
  return fetch(`http://musicbrainz.org/ws/2/artist?query=${search}&fmt=json&offset=${page - 1}`, {
```